orbit-db-cli/src/commands/feed/create.ts

```diff
@@ -0,0 +1,31 @@
+import { Command, Flags } from '@oclif/core';
+import { startOrbitDB } from '../../services/start-OrbitDB';
+import { stopOrbitDB } from '../../services/stop-OrbitDB';
+import { createDB } from '../../utils/create-DB';
```

Comment:
you could use imports starting with `@/` so it will be cleaner, see the [paths](https://www.typescriptlang.org/tsconfig#paths) property and [this example](https://nextjs.org/docs/app/building-your-application/configuring/absolute-imports-and-module-aliases)

Reply:
good remark, done !